### PR TITLE
Discord updates

### PR DESCRIPTION
**Pull request by Nathan-Notifi**
This is a doozy.

**So far,** this should allow users to toggle on for Discord and then generate a confirmation url that opens in a new window.

When toggled off, it should hide the discord field in the user info panel.
If a user has no discord targets, a random one should be created.

If active, the verify link should be be hidden.

### CODE REVIEW

File: packages/notifi-core/lib/operations/CreateDiscordTarget.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { DiscordTarget, Operation } from '../models';
+
+/**
+ * Input param for creating an DiscordTarget
+ *
+ * @remarks
+ * This describes the values to use for email target
```
Comment (eric-notifi) [inline packages/notifi-core/lib/operations/CreateDiscordTarget.ts:7]: Might want to change it to `Discord` target

File: packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts [hunk 2]
```diff
@@ -237,6 +290,7 @@ export const useNotifiSubscribe: ({
       // Mockup info for demo preview card
       setEmail(defaultDemoConfigV1.name);
       setPhoneNumber('+101234567890');
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
       setTelegramId(defaultDemoConfigV1.id!);
```
Comment (eric-notifi) [inline packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts:294]: Adding the dummy discord for preview card might be a good idea: `setDiscrodId(defaultDemoConfigV1.id!)`

**Review by eric-notifi — COMMENTED** [review]
LGTM, Just left few small points.

File: packages/notifi-core/lib/operations/CreateDiscordTarget.ts (hunk 1 above)
Comment (Nathan-Notifi) [reply packages/notifi-core/lib/operations/CreateDiscordTarget.ts:7]: great catch

File: packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts (hunk 2 above)
Comment (Nathan-Notifi) [reply packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts]: yeah, i added a dummy code for the telegram id so avoid the nonnull assertion as well

**Review by eric-notifi — APPROVED** [review]
It looks good to me.

File: packages/notifi-axios-adapter/lib/queries/getDiscordTargetsImpl.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  discordTargetFragment,
+  discordTargetFragmentDependencies,
+} from '../fragments';
+
+const DEPENDENCIES = [
+  ...discordTargetFragmentDependencies,
+  discordTargetFragment,
+];
+
+const MUTATION = `
```
Comment (eric-notifi) [inline packages/notifi-axios-adapter/lib/queries/getDiscordTargetsImpl.ts:17]: Not sure if it should be called `QUERY` ?

Comment (Nathan-Notifi) [reply packages/notifi-axios-adapter/lib/queries/getDiscordTargetsImpl.ts:17]: you're right. that is confusing. let me do a sweep and check

**Pull request merged**